### PR TITLE
DB output migration of RPCReadOutMapBuilder

### PR DESCRIPTION
**Pull request by tvami**
#### PR description:

Part of the migration for https://github.com/cms-AlCaDB/AlCaTools/issues/28

#### PR validation:

Code compiles, `scramv1 b runtests` runs 

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

N/A

### CODE REVIEW

**Comment by tvami** [comment]
@cmsbuild , please test

File: CondFormats/RPCObjects/test/RPCReadOutMapBuilder.cc [hunk 1]
```diff
@@ -73,7 +72,7 @@ void RPCReadOutMapBuilder::endJob() {
 void RPCReadOutMapBuilder::beginJob() {
   cout << "BeginJob method " << endl;
   cout << "Building RPC Cabling" << endl;
-  cabling = new RPCReadOutMapping("My map V-TEST");
+  RPCReadOutMapping cabling;
```
Comment (yuanchao) [inline CondFormats/RPCObjects/test/RPCReadOutMapBuilder.cc:75]: probably better to keep the original argument as the following?
`RPCReadOutMapping cabling("My map V-TEST");`

Comment (tvami) [reply CondFormats/RPCObjects/test/RPCReadOutMapBuilder.cc:75]: It's just a test naming, for sure doesn't loose functionality, and when I tried it didnt work immediately. If the owners of the code want to change it they can do it in a follow-up somewhen in the future.

**Comment by ggovi** [comment]
+db

**Comment by malbouis** [comment]
+alca

**Comment by qliphy** [comment]
+1

**Pull request merged**